tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -63,6 +66,699 @@ public function test_wp_enqueue_script() {
 		$this->assertSame( '', get_echo( 'wp_print_scripts' ) );
 	}
 
+	/**
+	 * Test standalone and non standalone inline scripts in the 'after' position of a single main script.
+	 *
+	 * @ticket 12009
```

Comment:
Add covers.

Reply:
@spacedmonkey I've opened a separate base PR to cover the addition of `@covers` to doc blocks, as well as assertion failure messages.

As discussed briefly in the WordPress performance slack channel, it's not clear _how_ deep one should go when referencing the functions and methods covered by a test, as some tests have very visually obviously call certain functions/methods, while others do not but _do_ rely on the utmost functioning of some underlying functions/methods to work, such as `WP_Scripts::do_item` and quite often (in our case) `WP_Scripts::get_eligible_loading_strategy`.

Nevertheless, I've tried to be as thoughtful as possible as to which to include and which not to include, perhaps you can take a look at the PR at https://github.com/10up/wordpress-develop/pull/61 and share your opinions there?

cc @joemcgill @kt-12 you may want to look at this as well.

Reply:
Thanks for the ping @10upsimon!

The key with `@covers` annotations is that they shouldn't list all functions/methods called in a test method, but only indicate _the function(s) under test_. Which function is being **tested** in a given unit test? Which function(s) are being **tested** in a given integration test?

For example:

```php
/**
 * Test set_option().
 *
 * @covers ::set_option
 */
public function test_set_option() {
    // Do some things related to set_option() before calling it.

    set_option( 'my_option', 'my_value' );
    $actual = get_option( 'my_option' );
    reset_option( 'my_option' );

    $this->assertSame( 'my_value', $actual );
}

/**
 * Test get_option().
 *
 * @covers ::get_option
 */
public function test_get_option() {
    set_option( 'my_option', 'my_value' );

    // Do some things related to get_option() before calling it.

    $actual = get_option( 'my_option' );
    reset_option( 'my_option' );

    $this->assertSame( 'my_value', $actual );
}
```
Bear in mind that this example is just for illustration, there is no `(re)set_option()` function 😄 It doesn't matter that `get_option()` and `reset_option()` are called in `test_set_option()`, coverage reports should only consider `set_option()` to be covered here. The reverse applies to the `test_get_option()` of course.